storage/jellyfish-merkle/src/nibble_path/mod.rs

```diff
@@ -146,7 +146,7 @@ impl NibblePath {
 
     /// Get the i-th bit.
     fn get_bit(&self, i: usize) -> bool {
-        assert!(i / 4 < self.num_nibbles);
+        assert!(i < self.num_nibbles * 4);
```

Comment:
I prefer the original code to avoid any overflow risk.

Reply:
We know self.num_nibbles<=64
multiplication is better than division